config/localization/en/app.json

```diff
@@ -2543,9 +2543,10 @@
       "NO_FUNDS_TO_RECLAIM": "There are no funds available for you to reclaim at this time.",
       "RECLAIM_FUNDS_SIGNING": "Upon reclaiming, you will be prompted to sign {NUM_TRANSACTIONS} transaction(s) to return the funds previously used as collateral for isolated margin trading."
     },
-    "SURGE_BANNER": {
-      "TITLE": "dYdX Surge: $3M Rewards Up for Grabs.",
-      "END_DATE": "Ends August 31"
+    "REBATE_BANNER": {
+      "TITLE": "This September: Get 50% of Trading Fees Back",
+      "SUBTITLE": "Paid out in October",
+      "BUTTON_TEXT": "Get Rebate"
```

Comment:
@fionn223 We should make the keys a little more descriptive, because otherwise the generated string_keys will only use STRING_KEY.TITLE (very vague).

I suggest prefixing them with REBATE_BANNER i.e.

REBATE_BANNER_TITLE, REBATE_BANNER_SUB, REBATE_BANNER_CTA